sormas-backend/src/test/java/de/symeda/sormas/backend/ArchitectureTest.java

```diff
@@ -291,6 +292,14 @@ private void assertFacadeEjbAnnotated(Class<?> facadeEjbClass, AuthMode authMode
 
 			methodChecks.check(classes);
 		}
+
+		// don't use RolesAllowed anymore, until performance problem is fixed
```

Comment:
This code block only checks EJBs that are given by a test. I would rather check all classes in the module annotated with `@Singleton`/`@Stateless` that there is no usage of `@RolesAllowed`.

Reply:
Guess there is no harm in simply not allowing it for all classes and methods in the package.